packages/terra-form-select/tests/wdio/select-spec.js

```diff
@@ -1734,17 +1732,14 @@ Terra.describeViewports('Select', ['tiny'], () => {
       });
 
       Terra.it.validatesElement('select-closed', { selector: '#root' });
+      after(() => browser.click('#root'));
```

Comment:
Can you please provide a description/explanation of these test changes? Why are they necessary, what is causing them to be necessary, and what happens/breaks if they are removed?

Reply:
Looks like this hook is not required hence removed 3c82c03